Issues with Let's Encrypt certificates

Hi:

We are using cabot to monitor our sites, and we have one them that has a SSL certificate obtained using Let's Encrypt. Since we enabled the certificate, cabot can't get the site status because it thinks that the certificate is not valid. The relevant error is:

> Request error occurred: [Errno 1] _ssl.c:504: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed

So I entered manually in the virtualenv, and updated the requests library, forcing the install of the [security] extra: 

> $ pip install --upgrade requests[security]

And now the alert is gone.

Perhaps the version of requests library required by cabot could be updated (currently it requires requests==2.7.0 and the 2.9.1 version is already out. Moreover the [security] extras should also be installed (which is present from requests 2.4.1) and provides additional security checks when making connections.


By the way, thank you for this excelent tool. This connected with the telegram plugin ( https://github.com/codesyntax/cabot_alert_telegram) makes our lives easier :wink:

--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@ setup(
         'psycopg2==2.5.1',
         'pytz==2014.10',
         'redis==2.9.0',
-        'requests==2.7.0',
+        'requests==2.9.1',
         'six==1.5.1',
         'twilio==3.4.1',
         'wsgiref==0.1.2',

